config.go

```diff
@@ -14,6 +14,8 @@ type config struct {
 	ClusterKey    []byte   `id:"cluster-key" desc:"shared key for cluster membership; must be 32 bytes base64 encoded; will be generated if not provided"`
 	Join          []string `desc:"comma separated list of hostnames or IP addresses to existing cluster members; if not provided, will attempt resuming any known state or otherwise wait for further members."`
 	Init          bool     `desc:"whether to explicitly (re)initialize the cluster; any known state from previous runs will be forgotten"`
+	Banner        string   `id:"banner" desc:"a string appended to /etc/hosts banner to discriminate between multiple running clusters"`
+	StatePath     string   `id:"state-path" desc:"path where the cluster state is stored; optional, depends on cluster name if not set"`
```

Comment:
I'm not sure if we need to expose these as settings explicitly. 
How about adding a cluster name setting and explicitly mentioning it's potential use?
```suggestion
	ClusterName        string   `id:"cluster-name" desc:"identifier for the wesher cluster; can be used to peer with multiple clusters" default:"default"`
```

Reply:
That was my first intention and implementation. However, one still needs to specify different ports, overlay networks, etc. so configuration is not not as trivial as setting the cluster name, plus one needs to carry the cluster name all the way to `state.save` and `loadState`.

I will reapply my commits on the matter so you can compare as well and we'll follow your preference.

Reply:
Now that I think about it a bit more, I wonder if we can't get rid of `ClusterName` as well.
As you mentioned, `ClusterPort` already need to be set for multiple instances to coexist. I'm not sure about `OverlayNet` (it's theoretically possible to have two clusters on the same net, as long as the hostnames are disjoint; maybe not the best idea, but possible).

So maybe we can just use these existing settings to determine banner and state file?

Reply:
The question for me seems to be: "what do I need to do, to run multiple wesher instances?"
And the answer should be "use different cluster ports (and probably overlay nets)"

We don't need to give the user the footguns for setting banner and state file path. These are implementation details that don't help him and would only mean more settings to change.

Does that make sense to you?

Reply:
Interface should also be different, since private key is per wireguard interface. Then so should be the wireguard port.

Maybe it makes more sense to use the interface name? It would also make things more humanly readable, you'll find your cluster state in `/var/lib/wesher/mywg.json` and `/etc/hosts` will clearly state which interface is responsible for a host.

Reply:
Yes, that makes more sense.
Ideally we'd als document the fact that multiple instances need separate:
- wg interface
- wg port
- cluster port
- (probably) overlay net

Reply:
OK. Then do we ensure at least some backward compatibility, and save to the current path if no interface name is specified?

We will break backward compatibility anyway for users that already specify an interface name.

Reply:
Not if we use the old path as a fallback on load.
- try to load the new path
- if not found try the old one
- save to the new one

Reply:
That should be okay indeed. Will do, thanks!